Examples/KlaviyoSwiftExamples/SPMExample/SPMExample.xcodeproj/project.pbxproj

```diff
@@ -782,8 +782,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/klaviyo/klaviyo-swift-sdk";
 			requirement = {
-				kind = upToNextMinorVersion;
-				minimumVersion = 3.0.0;
+				branch = master;
```

Comment:
updated to master since there were changes that were not present in earlier released version and this was causing a compiler issue.